packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -83,49 +110,53 @@ export class TreeConfiguration<TSchema extends ImplicitFieldSchema = ImplicitFie
 }
 
 /**
- * An editable view of a (version control style) branch of a shared tree.
+ * An editable view of a branch of a shared tree based on some schema.
+ *
+ * This schema--known as the view schema--may or may not align the stored schema of the document.
+ * Information about discrepancies between the two schemas is available via {@link TreeView.compatibility|compatibility}.
```

Comment:
Last I checked, intellisense wants spaces in aliased `@link`s. Does this work okay? If not, you may want:
```suggestion
 * Information about discrepancies between the two schemas is available via {@link TreeView.compatibility | compatibility}.
```